### PR TITLE
An 2811/near social

### PR DESCRIPTION
**Pull request by forgxyz**
Adds base models for NEAR Social schema
 - decoded events
 - addkey actions
 - profile edits

### CODE REVIEW

File: models/silver/social/silver_social__addkey.sql [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        A.block_timestamp,
+        A.allowance,
+        r.signer_id,
+        A._source,
+        A._partition_by_block_number,
+        A._load_timestamp
+    FROM
+        combine A
+        LEFT JOIN receipts r
+        ON SPLIT(
```
Comment (eric-laurello) [inline models/silver/social/silver_social__addkey.sql:94]: The join on a function can be super slow. I don't know if moving it up to a stand alone column in the CTE would make it better, but it's worth trying.

File: models/silver/social/silver_social__receipts.sql [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        {% if target.name == 'manual_fix' or target.name == 'manual_fix_dev' %}
+            {{ partition_load_manual('no_buffer') }}
+        {% else %}
+            {{ incremental_load_filter('_load_timestamp') }}
+        {% endif %}
+        AND (LOWER(signer_id) = 'social.near'
+        OR LOWER(receiver_id) = 'social.near')
+)
+SELECT
+    *
```
Comment (eric-laurello) [inline models/silver/social/silver_social__receipts.sql:24]: Can you replace one of the select * with the column names?

File: models/silver/social/silver_social__addkey.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+
+version: 2
+
+models:
+  - name: silver_social__addkey
+    description: |-
+      All AddKey events for the contract social.near, which indicate a wallet or user authorizing use of the contract as a high-level proxy for interest in the platform.
+
```
Comment (eric-laurello) [inline models/silver/social/silver_social__addkey.yml:8]: Are there any tests at all that need to be added?

File: models/silver/social/silver_social__receipts.sql [hunk 4]
```diff
@@ -0,0 +1,26 @@
+{{ config(
+    materialized = 'incremental',
+    unique_key = 'action_id',
```
Comment (eric-laurello) [inline models/silver/social/silver_social__receipts.sql:3]: Got a failure that action_id doesnt exist in the table.

Comment (forgxyz) [reply models/silver/social/silver_social__receipts.sql:3]: Whoops, bad copy/paste

**Review by eric-laurello — APPROVED** [review]

**Pull request merged**